src/main.cpp

```diff
@@ -1216,6 +1216,7 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
         // if it comes at all.
         CXVersionMessage xver;
         xver.set_u64c(XVer::BU_LISTEN_PORT, GetListenPort());
+        xver.set_u64c(XVer::BU_MSG_CHECKSUM, 1); // we will ignore 0 value msg checksums
         pfrom->PushMessage(NetMsgType::XVERSION, xver);
```

Comment:
not sure I understand that comment...does it mean you're using checksum or not?

Reply:
It mean that other nodes can send us a checksum with value 0 (and we'll skip doing the checksum).  How can I make the comment better?